src/test/java/com/thebois/models/world/structures/HouseTests.java

```diff
@@ -23,30 +34,206 @@ public void getPositionSetInConstructor() {
     }
 
     @Test
-    public void getTypeEqualsHouse() {
+    public void getTypeReturnsCorrectStructureType() {
         // Arrange
         final Position position = new Position(123, 456);
-
-        // Act
         final IStructure structure = StructureFactory.createStructure(
             StructureType.HOUSE,
             position);
+        // Act
+
+        StructureType returnedType = structure.getType();
 
         // Assert
-        assertThat(structure.getType()).isEqualTo(StructureType.HOUSE);
+        assertThat(returnedType).isEqualTo(StructureType.HOUSE);
     }
 
     @Test
-    public void deepCloneableIsEqualToOriginal() {
+    public void housesAreEqualWhenSameInstance() {
+        // Arrange
+        final Position positionA = new Position(123, 456);
+        final Position positionB = new Position(456, 123);
+
+        // Act
+        final IStructure structureA = StructureFactory.createStructure(
+            StructureType.HOUSE,
+            positionA);
+        final IStructure structureB = StructureFactory.createStructure(
+            StructureType.HOUSE,
+            positionB);
+
+        // Assert
+        assertThat(structureA).isNotEqualTo(structureB);
+    }
+
+    private static Stream<Arguments> tryDeliverItemReturnsExpectedValueSource() {
```

Comment:
style

I can see why you couldn't think of a good name for this method, since it returns four different arguments grouped together. But still, it's not a good idea to just use the name of the test and append "Source". These argument-streams are supposed to be able to be used in multiple different tests.

Reply:
I changed it, take a look

Reply:
Much better, good job!